### ISSUE TITLE
Publish on NPM fix already resolved

### ISSUE THREAD

**Issue by pantonio88** [wi3land/ionic-appauth#189]
On NPM repository are missing the fix developed in the last 3 months.

**Comment by pantonio88** [comment]
The last version published on NPM has issues while trying to acquire a new valid token using the refresh token.
In the repository source code, instead, the fix is already implemented.

**Comment by Dhug95** [comment]
I agree. It would be very useful so I can upgrade Capacitor version of my company project from 3 to 4.

**Comment by mraible** [comment]
If I had permission to publish lysis library, I would publish a new version. @wi3land 

Can you give me permission, or are you able to do a new release soon?

**Comment by pantonio88** [comment]
@wi3land: can you give the permission requested by @mraible ?

At this moment, the package published on NPM repository does not successfully manage the refresh token.

**Comment by leonluc-dev** [comment]
Seconded. It would be nice to have a up-to-date npm package.
It seems wi3land has been inactive on Github since september. Hopefully they are still around.

As for a temporary workaround, you can reference the GitHub commits directly in your NPM references (requires npm v.1.165 or up).
For example, the most recent commit (0d286fc) can be included using: 

  ```
"dependencies": {
    "ionic-appauth": "wi3land/ionic-appauth#0d286fc",
  }
```

**Comment by richardkshergold** [comment]
"It seems wi3land has been inactive on Github since september. Hopefully they are still around." - I'm not sure if they are -  will anyone else be maintaining this going forwards?

**Comment by mrogunlana** [comment]
Bump @mraible are you available to update the npm package?

**Comment by mraible** [comment]
@mrogunlana I don't have permission to publish the npm package. Only @wi3land has that.

**Comment by mraible** [comment]
@mrogunlana I now have permission to publish so hopefully we can get a release out soon.

**Comment by mraible** [comment]
See https://github.com/wi3land/ionic-appauth/releases/tag/v2.0.0.
